hypervisor/arch/x86/guest/vmsr.c

```diff
@@ -407,11 +407,11 @@ int32_t rdmsr_vmexit_handler(struct acrn_vcpu *vcpu)
 	case MSR_IA32_MTRR_FIX4K_F0000:
 	case MSR_IA32_MTRR_FIX4K_F8000:
 	{
-#ifdef CONFIG_MTRR_ENABLED
-		v = read_vmtrr(vcpu, msr);
-#else
-		err = -EACCES;
-#endif
+		if (!vm_hide_mtrr(vcpu->vm)) {
+			v = read_vmtrr(vcpu, msr);
+		} else {
+			err = -EACCES;
```

Comment:
If an MSR is not existed, and the guest access it, what happens? I thought it will trigegr #GP fault. Can u check the SDM?

Reply:
Yes. Specifying a reserved or unimplemented MSR address in ECX will also cause a
general protection exception.